locale/fi/LC_MESSAGES/django.po

```diff
@@ -882,49 +887,93 @@ msgstr "Lisää uusia määräajan muutoksia"
 msgid "Submit"
 msgstr "Lähetä"
 
-#: deviations/templates/deviations/list_dl.html:23
+#: deviations/templates/deviations/list_dl.html:22
+#: deviations/templates/deviations/remove_dl.html:6
+#: deviations/templates/deviations/remove_dl.html:12
+#: deviations/templates/deviations/remove_dl.html:19
+msgid "Remove deadline deviations"
+msgstr "Ei yhtään määräajan muutosta."
+
+#: deviations/templates/deviations/list_dl.html:27
 msgid "Deadline deviations"
 msgstr "Määräajan muutokset"
 
-#: deviations/templates/deviations/list_dl.html:27
+#: deviations/templates/deviations/list_dl.html:31
+#: deviations/templates/deviations/override.html:20
 msgid "Submitter"
 msgstr "Palauttaja"
 
-#: deviations/templates/deviations/list_dl.html:28
+#: deviations/templates/deviations/list_dl.html:32
+#: deviations/templates/deviations/override.html:21
 #: edit_course/templates/edit_course/edit_content.html:149
 #: exercise/templates/exercise/_user_results.html:81
 msgid "Exercise"
 msgstr "Tehtävä"
 
-#: deviations/templates/deviations/list_dl.html:29
+#: deviations/templates/deviations/list_dl.html:33
 msgid "Extra minutes"
 msgstr "Ylimääräiset minuutit"
 
-#: deviations/templates/deviations/list_dl.html:30
+#: deviations/templates/deviations/list_dl.html:34
 msgid "Without late penalty"
 msgstr "Ilman myöhästymissakkoa"
 
-#: deviations/templates/deviations/list_dl.html:31
+#: deviations/templates/deviations/list_dl.html:35
 #: edit_course/templates/edit_course/usertag_list.html:32
 msgid "Actions"
 msgstr "Toiminnot"
 
-#: deviations/templates/deviations/list_dl.html:51
+#: deviations/templates/deviations/list_dl.html:55
+#: deviations/templates/deviations/override.html:42
 msgid "No deadline deviations."
 msgstr "Ei yhtään määräajan muutosta."
 
-#: deviations/views.py:43
+#: deviations/templates/deviations/override.html:10
+msgid ""
+"Some of the deviations already existed! Please select the deviations you "
+"want to override!"
+msgstr "Osa määräajan muutoksista oli jo olemassa!"
+"Valitse ylikirjoitettavat muutokset!"
+
+#: deviations/templates/deviations/override.html:16
+msgid "Select deviations to be overridden"
+msgstr "Valitse ylikirjoitettavat muutokset"
+
+#: deviations/templates/deviations/override.html:22
+msgid "Extra time before (minutes)"
+msgstr "Lisä aika ennen (minuuteissa)"
```

Comment:
"extra time before": do users understand that "before" means the previous old value that would be overwritten? Is there a better word for this? The wording may mislead people into thinking that this is some extra time given before something else.

"Lisä aika" should be "lisäaika".

Reply:
Words to replace "before": "Current extra time"